backend/migrations/versions/a5d22b31faab_add_resident_table.py

```diff
@@ -48,7 +50,6 @@ def upgrade():
 
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.drop_constraint("check_date_left_valid", "residents", type_="check")
```

Comment:
is there a reason this was deleted

Reply:
i don't recall doing anything of the sort

Reply:
it looks like it should still be there so if you could add it back that would be 🙏